include/boost/geometry/strategies/geographic/parameters.hpp

```diff
@@ -24,6 +26,23 @@ namespace boost { namespace geometry { namespace strategy
 
 struct andoyer
 {
+    //TODO: this should be replaced by an andoyer direct formula
```

Comment:
Right, implementing andoyer direct formula would be preferable to using thomas.

Reply:
This needs some research. I propose to do this in a separate PR. It is expected that using thomas direct it should be a bit slower and with small deviations in the computed result compared to the future andoyer direct formula.